verify.js

```diff
@@ -31,6 +31,22 @@ const noDuplicates = Object.keys(db)
       let numberOfUniqueIdentifiers = (new Set(identifiers)).size;
       if (numberOfItems !== numberOfUniqueIdentifiers) {
         console.error(`Failed to verify '${list}': '${numberOfItems - numberOfUniqueIdentifiers}' duplicate Identifier(s) found.`);
+        
+        let identifierCount = {};
+        for(let i = 0; i < identifierCount.length; i++){ 
+          if (identifierCount[identifiers[i]]) {
+            identifierCount[identifiers[i]] += 1;
+          } else {
+            identifierCount[identifiers[i]] = 1;
+          } 
+        }
+
+        for (let identifier in identifierCount) {
+          if (identifierCount[identifier] >= 2) {
+            console.error(identifier + " found " + duplicateCounter[identifier] + " times.");
+          }
+        }
+
```

Comment:
Unfortunately your change did not report the duplicate, only that there is one.
 
Though this doesn't tell you how many times a duplicate is found, this is a simpler solution:
```suggestion
      const duplicates = identifiers.filter((item, index) => identifiers.indexOf(item) !== index);
      if (duplicates.length > 0) {
        console.error(`Failed to verify '${list}', duplicate identifiers found: \n${duplicates.join('\n')}`);
```